poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.4 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
not a big one, but we need to ensure the same poetry version as the one in the Dockerfile is used to update the lockfile (currently 1.8.4 - https://github.com/ONSdigital/dis-wagtail/blob/main/Dockerfile#L76)